.codecov.yml

```diff
@@ -0,0 +1,18 @@
+coverage:
+  range: 80..100
+  round: down
+  precision: 2
+
+  status:
+    project:                   # measuring the overall project coverage
+      default:                 # context, you can create multiple ones with custom titles
+        enabled: yes           # must be yes|true to enable this status
+        target: auto           # specify the target coverage for each commit status
+        treshold: 5%           #   option: "auto" (must increase from parent commit or pull request base)
```

Comment:
You have threshold (without a type one line below). Anyway for me looks good now, but please set threshold to 1% only.